### PR TITLE
Implement OnchainKit features

### PR DESCRIPTION
**Pull request by mine00clone**
## Description
I implemented the following features using OnchainKit.

- Display avatars
- If ENS exists, the ENS is shown
- A check mark will appear if the account has been verified on Coinbase

## Screenshot

スクリーンショット 2024-06-25 235449

スクリーンショット 2024-06-26 002124

### CODE REVIEW

**Comment by taijusanagi** [comment]
LGTM!

File: app/src/app/providers.tsx [hunk 1]
```diff
@@ -32,7 +35,14 @@ export function Providers({ children }: { children: React.ReactNode }) {
             accentColor: "#22CC02",
           })}
         >
-          <ApolloProvider client={apolloClient}>{children}</ApolloProvider>
+          <ApolloProvider client={apolloClient}>
+            <OnchainKitProvider
+              apiKey={NEXT_PUBLIC_COINBASE_API_KEY}
```
Comment (taijusanagi) [inline app/src/app/providers.tsx:40]: Is this api key really required?

File: app/src/components/creatorIdentity.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { COINBASE_VERIFIED_ACCOUNT_SCHEMA_ID } from "@/config/const";
+import { Avatar, Badge, Name, Identity, Address } from '@coinbase/onchainkit/identity';
+
+
+const CreatorIdentity = ({ address }: { address: string }) => {
+  return (
+    <Identity
+      className="bg-transparent"
+      schemaId={COINBASE_VERIFIED_ACCOUNT_SCHEMA_ID}
```
Comment (taijusanagi) [inline app/src/components/creatorIdentity.tsx:9]: I think schemaId is better to be set at provider like this
https://github.com/coinbase/onchainkit?tab=readme-ov-file#components

File: app/tailwind.config.ts [hunk 3]
```diff
@@ -1,13 +1,63 @@
 import type { Config } from "tailwindcss";
+import path from 'node:path'; 
 
 const config: Config = {
   content: [
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/**/*.{ts,tsx}",
```
Comment (taijusanagi) [inline app/tailwind.config.ts:9]: Is this line required? It looks duplicated with the above 3 lines.

File: app/tailwind.config.ts [hunk 4]
```diff
@@ -1,13 +1,63 @@
...
+    path.join(path.dirname(require.resolve('@coinbase/onchainkit')), '**/*.js'), 
   ],
   theme: {
-    extend: {},
+    fontFamily: {
+      sans: ['Inter', 'sans-serif'],
+    },
+    extend: {  
+      fontFamily: {  
+        display: 'DM Sans, sans-serif', // Set to font of your choice
```
Comment (taijusanagi) [inline app/tailwind.config.ts:18]: Current EIS using Robot font, can you modify to set the font properly?

**Review by taijusanagi — COMMENTED** [review]
Hi, I reviewed the PR! Can you please check it?

File: app/src/components/creatorIdentity.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      address={address as `0x${string}`}
```
Comment (taijusanagi) [inline app/src/components/creatorIdentity.tsx:10]: You can use Address type in viem

File: app/src/app/providers.tsx (hunk 1 above)
Comment (mine00clone) [reply app/src/app/providers.tsx]: I've checked that just fetching ENS and verification status works without an API key. I'll remove it.

File: app/src/components/creatorIdentity.tsx (hunk 2 above)
Comment (mine00clone) [reply app/src/components/creatorIdentity.tsx]: You're right. I will move the schemaId setting to the top level.

File: app/tailwind.config.ts (hunk 3 above)
Comment (mine00clone) [reply app/tailwind.config.ts]: My apologies. You are right; they're duplicated. I'll delete that part.

File: app/tailwind.config.ts (hunk 4 above)
Comment (mine00clone) [reply app/tailwind.config.ts]: Sure. I'll replace it to Robot.

File: app/src/components/creatorIdentity.tsx (hunk 5 above)
Comment (mine00clone) [reply app/src/components/creatorIdentity.tsx]: OK, I'll fix it.

**Comment by mine00clone** [comment]
@taijusanagi 
I think all the problems you pointed out are now solved. Could you review them again?

**Pull request merged**